src/test/java/org/minions/devfund/noemiguzman/battleship/ShipTest.java

```diff
@@ -0,0 +1,216 @@
+package org.minions.devfund.noemiguzman.battleship;
+
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * class to test ship methods.
+ */
+public class ShipTest {
+    /**
+     * test get ship type.
+     */
+    @Test
+    public void testGetShipType() {
+        final int expectedLength = 8;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "expectedLength". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "expectedLength" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)